etna/collections/models.py

```diff
@@ -245,22 +245,22 @@ def get_datalayer_data(self, request: HttpRequest) -> Dict[str, Any]:
     @cached_property
     def related_articles(self):
         """Return a list of related pages for rendering in the related articles section
-        of the page. To add another page type, add it to the `exact_type` query below.
+        of the page. To add another page type, import it and add it to the related_page_types list.
         """
 
-        from wagtail.models import Page
-
         from etna.articles.models import ArticlePage, FocusedArticlePage
 
-        return (
-            Page.objects.exclude(pk=self.featured_article.pk)
-            .live()
-            .public()
-            .exact_type(ArticlePage, FocusedArticlePage)
-            .filter(pk__in=self.related_page_pks)
-            .order_by("-first_published_at")
-            .specific()
-        )
+        related_page_types = [ArticlePage, FocusedArticlePage]
+        page_list = []
+
+        for page_type in related_page_types:
+            pages = page_type.objects.exclude(pk=self.featured_article_id).filter(pk__in=self.related_page_pks).live().public().prefetch_related("teaser_image__renditions")
```

Comment:
NOTE: We'll want to keep the `select_related("teaser_image")` if going back to specific page type queries :)